tests/fixtures/device_specs.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Nit: It will probably be a good idea to add a document (README?) describing this file. Or, if se can use `jsonc` instead of `json`, we might be able to smuggle comments within the file.